lib/SILGen/SILGenDestructor.cpp

```diff
@@ -219,6 +220,119 @@ void SILGenFunction::destroyClassMember(SILLocation cleanupLoc,
   }
 }
 
+void findRecursiveLinks(ClassDecl *cd, llvm::SmallSetVector<VarDecl*, 4> &result) {
```

Comment:
Can you run git-clang-format on this? VarDecl * should have been fixed on this. Also generally people for these types of local functions will add static. Since it is a declaration, the doxygen should be three slashes above the function.